ansible/roles/mn_protx_diff/files/protx-diff.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+BLOCKCOUNT=$(dash-cli getblockcount)
+DIP3BLOCK=$(dash-cli getblockchaininfo | jq .bip9_softforks.dip0003.since)
+BASEBLOCK=$(shuf -i $DIP3BLOCK-$BLOCKCOUNT -n 1)
+ENDBLOCK=$(shuf -i $BASEBLOCK-$BLOCKCOUNT -n 1)
```

Comment:
Am I correct this is doing subtraction? I think the use of `protx diff` is using the actual heights such as `protx diff 100020 100000`

Reply:
This isn't subtraction, it's an argument to `shuf -i` which is `-i, --input-range=LO-HI   treat each number LO through HI as an input line`

Reply:
aah, okay :) sounds good